[2023-03-23T10:19:32.427+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gcs_2_bq_dag.move_yellow_tripdata_external_table scheduled__2023-03-21T00:00:00+00:00 [queued]>
[2023-03-23T10:19:32.440+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gcs_2_bq_dag.move_yellow_tripdata_external_table scheduled__2023-03-21T00:00:00+00:00 [queued]>
[2023-03-23T10:19:32.441+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-23T10:19:32.441+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2023-03-23T10:19:32.441+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-23T10:19:32.462+0000] {taskinstance.py:1300} INFO - Executing <Task(BigQueryCreateExternalTableOperator): move_yellow_tripdata_external_table> on 2023-03-21 00:00:00+00:00
[2023-03-23T10:19:32.470+0000] {standard_task_runner.py:55} INFO - Started process 93 to run task
[2023-03-23T10:19:32.475+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'gcs_2_bq_dag', 'move_yellow_tripdata_external_table', 'scheduled__2023-03-21T00:00:00+00:00', '--job-id', '1926', '--raw', '--subdir', 'DAGS_FOLDER/transfer_gcp_to_bq.py', '--cfg-path', '/tmp/tmphq0h5of4']
[2023-03-23T10:19:32.477+0000] {standard_task_runner.py:83} INFO - Job 1926: Subtask move_yellow_tripdata_external_table
[2023-03-23T10:19:32.494+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-03-23T10:19:32.520+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-03-23T10:19:32.562+0000] {task_command.py:388} INFO - Running <TaskInstance: gcs_2_bq_dag.move_yellow_tripdata_external_table scheduled__2023-03-21T00:00:00+00:00 [running]> on host 4b3eeaf199d3
[2023-03-23T10:19:32.692+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=move_yellow_tripdata_external_table
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-21T00:00:00+00:00
[2023-03-23T10:19:32.693+0000] {crypto.py:83} WARNING - empty cryptography key - values will not be stored encrypted.
[2023-03-23T10:19:32.693+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-03-23T10:19:33.653+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=gcs_2_bq_dag, task_id=move_yellow_tripdata_external_table, execution_date=20230321T000000, start_date=20230323T101932, end_date=20230323T101933
[2023-03-23T10:19:33.701+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-23T10:19:33.728+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
